citations-tool/tool/src/webapp/js/edit_nested_citations.js

```diff
@@ -260,6 +260,7 @@
             if ( !CKEDITOR.instances[sectionInlineEditor] ) {
                 CKEDITOR.inline( sectionInlineEditor, {
                     startupFocus: true,
+                    forcePasteAsPlainText : true,
```

Comment:
Just discussing in office. It would be ideal to allow through the limited styles we accept but this isn't simple/quick.